.github/workflows/publish.yaml

```diff
@@ -0,0 +1,56 @@
+name: Publish
```

Comment:
Figured out a better way to handle releases. 

With this workflow, we can manage releases by creating a dummy release [here](https://github.com/appannie/ab-testing/releases)

And the workflow will replace the dummy release with a real one, with proper release notes and packaging. 

The dummy releases are identified by their tag names. currently 3 possible values: `patch` `minor` and `major`.

@SBoudrias